SwipeActions.js

```diff
@@ -101,22 +101,27 @@ export default class SwipeActions extends React.Component {
   }
 
   render() {
+    let actionViewRight = 0;
     let actionViews = this.state.width === 0 ?
         null : this.props.actions.map((action, index) => {
-      return (
+      let customStyle = (action.containerStyle) ? action.containerStyle : {};
```

Comment:
Pass `action.containerStyle` directly into the style prop array which can contain null/undefined values just fine.